.github/workflows/ci.yml

```diff
@@ -205,4 +205,4 @@ jobs:
       ts-version: 'v1.10.2'
       java-version: 'v1.24.2'
```

Comment:
Are these versions supposed to be in sync with the versions earlier in the file?

Reply:
I am unsure, but they look in sync to me